app/src/main/res/values/strings.xml

```diff
@@ -235,31 +235,31 @@
     <string name="update_now">Update now</string>
     <string name="status_updating">Updating…</string>
     <string name="malware_detection_learn_more">Learn more about the malware detection feature</string>
-    <string name="private_dns_message_notice">Private DNS prevents PCAPdroid from inspecting the DNS traffic. You can disable it from the Android network settings.</string>
+    <string name="private_dns_message_notice">Private DNS prevents PCAPdroid from inspecting the DNS traffic. You can turn it off from the Android network settings.</string>
     <string name="private_dns_hinders_detection">Private DNS hinders detection</string>
-    <string name="decryption_no_filter_warn">When performing TLS decryption you should select a target app to avoid breaking Internet connectivity</string>
+    <string name="decryption_no_filter_warn">Select a target app when decrypting TLS to avoid breaking Internet connectivity</string>
```

Comment:
```suggestion
    <string name="decryption_no_filter_warn">Select a target app when decrypting TLS to avoid losing Internet connectivity</string>
```